pages/database-management/authentication-and-authorization/auth-system-integrations.mdx

```diff
@@ -409,3 +409,80 @@ allowed to log in because her role (moderator) already exists. Carol and Dave
 won't be allowed to log in because their role (administrator) doesn't exist.
 
 </Steps>
+
+## OpenID Connect (OIDC)
+
+Memgraph also supports authentication and authorization using OIDC (OIDC + OAuth 2.0) protocol with a
+built-in auth module that is packaged with Memgraph Enterprise.
+
+The module currently supports the following SSO providers:
+- Microsoft Entra ID
+- Okta using a custom authorization server
+
+For SSO provider setup see [this](/data-visualization/user-manual/single-sign-on).
+
+### Module requirements
+
+The module is written in Python 3 and requires the following libraries:
+ - `jwt` - used to validate tokens.
+ - `requests` - used to retrieve public keys for validating tokens.
+
+The required packages are already installed in the docker image or native Memgraph if you followed the installation procedure correctly.
+
+### Module configuration
+
+The module is located at `/etc/memgraph/auth_module/oidc.py`.
+The module has no configuration file. You only have to run Memgraph with some environmental variables.
+
+
+#### MS Entra ID
+For using OIDC with Microsoft Entra ID you have to set the following environmental variables:
+
+```bash
+MEMGRAPH_SSO_ENTRA_ID_OIDC_TENANT_ID=
+MEMGRAPH_SSO_ENTRA_ID_OIDC_CLIENT_ID=
+MEMGRAPH_SSO_ENTRA_ID_OIDC_ROLE_MAPPING=
+MEMGRAPH_SSO_ENTRA_ID_OIDC_USERNAME=
+```
+
+You can find the client and tenant ID in the app overview in Azure. For a more detailed explanation see [this](/pages/data-visualization/user-manual/single-sign-on.mdx).
+
+##### Role mapping
+Authorization is done by mapping the Entra ID roles from the roles field in the access token to the Memgraph roles. Make sure that each user only has one role on Entra ID.
+One example of role mapping is `entra.admin:memadmin; entra.user:memuser`.
+
+##### Username
+The username variable tells the OIDC module what to use as the username. It has the format `token-type:field`. 
+Token type can be `id` or `access` depending on whether you want to use a field from the access or the ID token for the username.
```

Comment:
Same note as above. Be consistent in the ordering of talking about things.

Reply:
Please explain more about the difference between "id" and "access". Perhaps link to an Okta page?